omegalambda/main/common/util/conversion_utils.py

```diff
@@ -0,0 +1,292 @@
+import numpy as np
+import re
+import datetime
+from typing import Tuple, Union, Optional
+
+from astropy import units as u
+from astropy.coordinates import SkyCoord, FK5, AltAz, get_sun, EarthLocation
+from astropy.time import Time
+
+from . import time_utils
+
+
+def convert_altaz_to_radec(azimuth: float, altitude: float, latitude: float, longitude: float,
+                           time: datetime.datetime, leap_seconds: float = 0, refraction = True) -> Tuple[float, float]:
+    """
+    Convert horizontal coordinates to celestial coordinates.
+    Formulas gathered from the following references.
+    References:
+        1.  K. Collins and J. Kielkopf, “Astroimagej: Imagej for astronomy,” (2013). Astrophysics source code library.
+            https://github.com/karenacollins/AstroImageJ.
+
+    Parameters
+    ----------
+    azimuth : FLOAT
+        Given azimuth of target, degrees.
+    altitude : FLOAT
+        Given altitude of target, degrees.
+    latitude : FLOAT
+        Latitude of observatory, degrees North.
+    longitude : FLOAT
+        Longitude of observatory, degrees East.
+    time : datetime.datetime object
+        Time to be converted.
+    leap_seconds : INT
+        Leap second offset between TAI and TT.
+    refraction : BOOL
+        Whether or not to correct for atmospheric refraction.
+
+    Returns
+    -------
+    ra : FLOAT
+        Calculated right ascension of target.
+    declination : FLOAT
+        Calculated declination of target.
+    """
+    if refraction:
+        pressure = 760
```

Comment:
You may want to have these set as global vars incase you ever want to change these all the values are at the top of the page.